@types/interfaces.d.ts

```diff
@@ -0,0 +1 @@
+export type ScrollerScrollDirection = 'horizontal' | 'vertical' | 'none' | null | undefined;
```

Comment:
Nit: since `null` is not a documented value for this and our code never sets this to `null` I think it could be left out from here. Even though it works the same as `undefined`. If the user uses the TS defs they can make sure to not use `null` and just use the expected/documented values.

Reply:
> I added null and undefined here instead of in the JSDoc, because they are explicitly mentioned in the docs as a valid scroll direction, so IMO they should be part of the type definition.

I just noticed this above (it wasn't showing as I started my review before you posted it). Is `null` really documented somewhere?

Reply:
Makes sense, done